easybuild/tools/toolchain/toolchain.py

```diff
@@ -647,41 +651,64 @@ def prepare(self, onlymod=None, silent=False, loadmod=True):
                 self.generate_vars()
                 self._setenv_variables(onlymod, verbose=not silent)
 
-        if build_option('use_compiler_cache'):
-            self.prepare_compiler_cache()
+        # consider f90cache first, since ccache can also wrap Fortran compilers
+        for cache_tool in [F90CACHE, CCACHE]:
```

Comment:
it's only used in one place now, and it has to be in a specific order here, so I don't see the point in making it available globally? it'll make it harder to see why it's in a particular order